wal/src/table_kv_impl/region.rs

```diff
@@ -691,20 +691,26 @@ impl<T: TableKv> BlockingLogIterator for TableLogIterator<T> {
             .context(manager::Decoding)?;
         let payload = self
             .log_encoding
-            .decode_value(current_iter.value(), decoder)
+            .decode_value(current_iter.value())
             .map_err(|e| Box::new(e) as _)
             .context(manager::Encoding)?;
-        let log_entry = LogEntry {
-            sequence: self.current_log_key.1,
-            payload,
-        };
+
+        // To unblock pr#119, we use the following to simple resolve borrow-check error.
+        // detail info: https://github.com/CeresDB/ceresdb/issues/120
+        let value_start = self.values.len();
+        self.values.extend_from_slice(payload);
```

Comment:
Is it necessary to keep previous bytes rather than drop them? How about
```rust
self.values = payload.to_owned();
// ...
&self.values;
```

Reply:
Make sense.